tldw_Server_API/app/core/Evaluations/embeddings_abtest_service.py

```
@@ -584,14 +608,24 @@ async def run_vector_search_and_score(
     user_id: str,
     arm_collections: list[dict[str, str]],
+    *,
+    manager_factory: Any | None = None,
 ) -> dict[str, dict[str, float]]:
     """Run vector-only search across arms and compute metrics, storing results in DB.
 
     Returns aggregate metrics per arm_id.
     """
+    search_mode = (config.retrieval.search_mode or "vector").lower()
+    if search_mode == "fts":
+        raise ValueError("fts search_mode is not supported by the embeddings A/B vector retrieval path.")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)